src/panel/direction/RouteStartEndTimes.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { getTimeFormatter } from 'src/libs/time';
+
+const RouteStartEndTimes = ({ start, end, className }) => {
+  if (!start || !end) {
+    return null;
+  }
+
+  const timeFormatter = getTimeFormatter({ hour: '2-digit', minute: '2-digit' });
+
+  return (
+    <div className={cx('u-bold', className)}>
+      {timeFormatter.format(new Date(start))} - {timeFormatter.format(new Date(end))}
```

Comment:
_Again that painful moment when we have to deal with timezones in Javascript :sob:_ 

This does not look correct to me. `new Date(start)` is a Date object that is no longer aware of the timezone offset returned by the server. As a result, the formatted time is implicitly converted into the user timezone, instead of the timezone localized at the start or end point. This is not what is expected, is it ?

I don't know any ideal solution using JS `Date` (see [this question](https://stackoverflow.com/q/17751698/5634396)).
In this use case though, stripping the offset part (e.g `+0300`) from the server response before calling `new Date()` could probably do the trick. The datetime will then be interpreted as being defined in the user timezone without any conversion, which is technically wrong but good enough for formatting purposes.

Reply:
Yes sorry I forgot that (or more probably my subconscience wanted to).
Let's just strip the offset as you propose, so we'll be in the same case as the opening hours.